server/@types/hmpo-form-wizard/index.d.ts

```diff
@@ -0,0 +1,283 @@
+// Copied from https://github.com/ministryofjustice/hmpps-strengths-based-needs-assessments-ui/blob/c1d2e0ac64ce2be0ea0a27919cd34f8882b2d5c0/server/%40types/hmpo-form-wizard/index.d.ts
+
+/* eslint-disable max-classes-per-file */
+declare module 'hmpo-form-wizard' {
+  import Express from 'express'
+
+  // These enums have to live here because of TS/Jest and Enums work..  ¯\_(ツ)_/¯
+  // Also this ESLint override because of how TS/Eslint works.
+  // eslint-disable-next-line no-shadow
```

Comment:
eslint is saying that you don’t need any of the `no-shadow` exemptions, but elsewhere you can just ignore the problem with `any` with
```javascript
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Reply:
ah, I was trying:
`// eslint-disable-next-line no-explicit-any`
and it wasn't working. Will try the above thank you.